Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:lts-alpine As development
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+FROM node:lts-alpine as production
```

Comment:
Same as above.

Reply:
Pinned it to node:16.14.2-alpine3.15 now.